extensions/amp-sidebar/1.0/component.js

```diff
@@ -238,14 +292,16 @@ export function SidebarToolbar({
   }, [mediaQuery, toolbarTarget, targetEl]);
 
   return (
-    <nav
-      ref={ref}
-      toolbar={mediaQueryProp}
-      toolbar-target={toolbarTargetProp}
-      {...rest}
-    >
-      {children}
-    </nav>
+    preactMode && (
+      <nav
+        ref={ref}
+        toolbar={mediaQueryProp}
+        toolbar-target={toolbarTargetProp}
+        {...rest}
+      >
+        {children}
+      </nav>
+    )
```

Comment:
In preact mode, we render the Toolbar in place here.  (This is displayed within the Sidebar element).  In amp mode we do not need to render anything in the sidebar since the `nav` element is already rendered into the sidebar via `slot`.

The cloned elements are handled in the effects and should happen in both preact and amp modes.